### DISCUSSION TITLE
Partial loss of labels within dictionary

### DISCUSSION THREAD

**Discussion by mengxingwu** [cnr-isti-vclab/TagLab#44] [General]
Hello, 
As I am labelling, I find part of my dictionary is blocked, only the labels I have used is shown.
I can't use other labels I haven't use. I've re-imported the default dictionary, but it didn't work.
How can I fix this problem ?

**Comment by maxcorsini** [comment]
What do you mean exactly with "blocked"? You cannot select them / assign them ?
